pandas/_libs/src/parser/tokenizer.h

```diff
@@ -31,9 +25,6 @@ See LICENSE for the license
 
 #include "khash.h"
 
-#define CHUNKSIZE 1024 * 256
```

Comment:
these are not used?

Reply:
No; there is a `DEFAULT_CHUNKSIZE`defined in parsers.pyx which appears used instead